spec/lib/triannon-client/triannon-client_spec.rb

```diff
@@ -55,12 +137,16 @@ def check_graph_has_statements(graph)
       # allow(@tc).to receive(@sitle).and_return(back_end)
       # expect(@tc.delete_annotation).to be true
     end
-    it 'should DELETE an open annotation that exists' do
-      expect( @tc.delete_annotation(@anno[:id]) ).to be_truthy
+    it 'deletes an open annotation that exists' do
+      anno = create_annotation
+      id = anno[:id]
+      tc = TriannonClient::TriannonClient.new
+      expect( tc.delete_annotation(id) ).to be true
     end
-    it 'should fail to DELETE an open annotation that does NOT exist' do
+    it 'fails to delete an open annotation that does NOT exist' do
```

Comment:
Yes, perhaps the 'returns <x>' is more informative, wheras the 'fails' might imply some kind of error or exception.